src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -62,10 +62,17 @@ public static class TextLoaderSaverCatalog
         /// <summary>
         /// Create a text loader <see cref="TextLoader"/> by inferencing the dataset schema from a data model type.
         /// </summary>
+        /// <typeparam name="TInput">Defines the schema of the data to be loaded. Use public fields or properties
+        /// decorated with <see cref="LoadColumnAttribute"/> (and possibly other attributes) to specify the column
+        /// names and their data types in the schema of the loaded data.</typeparam>
         /// <param name="catalog">The <see cref="DataOperationsCatalog"/> catalog.</param>
         /// <param name="separatorChar">Column separator character. Default is '\t'</param>
         /// <param name="hasHeader">Does the file contains header?</param>
-        /// <param name="dataSample">The optional location of a data sample. The sample can be used to infer column names and number of slots in each column.</param>
+        /// <param name="dataSample">The optional location of a data sample. The sample can be used to infer column
+        /// names and number of slots in each column. If <typeparamref name="TInput"/> does not contain any public
+        /// fields, we assume that the first column in the <paramref name="dataSample"/> is the Label column and
+        /// the second is the Features column. If there are more than two columns, the Features column is taken
```

Comment:
This is great! Could you add this point to the other overloads of this API as well? (the one taking the `Options` object, and the one taking the `Column` array directly)